fe2-android/app/src/test/java/com/github/dedis/popstellar/utility/handler/RollCallHandlerTest.java

```diff
@@ -89,8 +90,7 @@ public void setup() throws GeneralSecurityException, IOException, KeyException {
         });
 
     // Add the LAO to the LAORepository
-    laoRepository.getLaoById().put(lao.getId(), new LAOState(lao));
-    laoRepository.setAllLaoSubject();
+    laoRepository.updateLao(lao);
```

Comment:
so why the comment says "add", but the function name says "update" ?

Reply:
The updateLao function updates the Lao if present and adds it otherwise. To me the comment is ok but I can change it if you feel strongly about it.